### PR TITLE
Update image assets with new logos

### PR DESCRIPTION
**Pull request by zmb3**

### CODE REVIEW

**Review by codingllama — APPROVED** [review]
Nice!

File: assets/img/readme-header.png
Comment (zmb3) [inline assets/img/readme-header.png:1]: I suppose we could just remove this, doesn't seem to be linked anywhere anymore.

Comment (codingllama) [reply assets/img/readme-header.png:1]: Unless you wanna re-introduce it to the README?

Comment (codingllama) [reply assets/img/readme-header.png:1]: It might make the header too busy, though. LGTM whatever you prefer.

**Review by codingllama — APPROVED** [review]

**Review by r0mant — APPROVED** [review]

**Pull request merged**